# Request
int32 robot_id  # LEFT=0, RIGHT=1, DUAL=-1
geometry_msgs/Pose target_pose  # Target Cartesian pose
bool move_mode  # ABS=0, INCR=1
float64 velocity  # Cartesian velocity (mm/s)
float64 acceleration  # Cartesian acceleration (mm/s^2)
bool is_block  # TRUE for blocking, FALSE for non-blocking
---
# Response
bool success
string message
